services/frontend-service/src/ui/utils/store.tsx

```diff
@@ -344,16 +335,50 @@ export const searchCustomFilter = (queryContent: string | null, val: string | un
     }
 };
 
-// return app lock IDs
-export const useApplicationLockIDs = (): string[] =>
-    useOverview(({ environments }) =>
-        Object.values(environments)
-            .map((env) => Object.values(env.applications))
-            .flat()
-            .map((app) => Object.values(app.locks))
-            .flat()
-            .map((lock) => lock.lockId)
-    );
+export type AllLocks = {
```

Comment:
I remember having to change this once before because of the flat mapper error with the store. If it doesn't show anything on the console I would assume it's fine. :thinking:

Reply:
I created and deleted a couple locks, and I don't see anything in the console.